README.md

```diff
@@ -7,14 +7,15 @@
 Simple Rust library to generate X.509 certificates.
 
 ```Rust
-use rcgen::generate_simple_self_signed;
+extern crate rcgen;
```

Comment:
Let's skip the `extern crate rcgen;` line, which hasn't been needed in a long time (feel free to remove it from `lib.rs`, too).

Reply:
Done.